container-host-files/etc/hcf/config/role-manifest.yml

```diff
@@ -1334,6 +1290,90 @@ roles:
                 values:
                 - diego-access
             topologyKey: "beta.kubernetes.io/os"
+# - name: ceph-broker
```

Comment:
‽‽‽

Reply:
I didn't want to get rid of that piece, because we'll need to make it work anyway.
But I guess I'll stash it away in a different branch